modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/NavigationBar.js

```diff
@@ -13,46 +13,42 @@
  */
 
 import ClayButton from '@clayui/button';
-import ClayLayout from '@clayui/layout';
-import ClayLink from '@clayui/link';
 import ClayNavigationBar from '@clayui/navigation-bar';
-import classNames from 'classnames';
-import React from 'react';
+import ClayLink from '@sclayui/link';
+import React from 'resact';
 
-export default function NavigationBar({cssClass, inverted, navigationItems}) {
+export default function NavigationBar({inverted, navigationItems}) {
```

Comment:
I think we should pass `cssClass` as `className` to `<ClayNavigationBar>`.

That is:

```javascript
<clay:navigation-bar cssClass="foo" />
```

Would need to render 

```javascript
<ClayNavigationBar className="foo" />
```

That's the main pattern we have, that `cssClass` allows devs to customize classes in the main container.

But we might have issues here with this, maybe?

Reply:
I think we can try it